packages/nextjs-mf/src/plugins/NextFederationPlugin/index.ts

```diff
@@ -164,7 +164,7 @@ export class NextFederationPlugin {
       ],
       //@ts-ignore
       exposes: {
-        ...(this._extraOptions.skipSharingNextInternals ? {} : defaultExpose),
+        ...defaultExpose,
```

Comment:
This should remain here. Users may want to disable sharing defaults in next.

Reply:
@ScriptedAlchemy Isn't this redundant logic?
`this._extraOptions.skipSharingNextInternals ? `

https://github.com/module-federation/core/blob/88535f272088915aaad4a0266c2a5f40e2e59369/packages/nextjs-mf/src/plugins/NextFederationPlugin/index.ts#L149-L156